### PR TITLE
added dustLog method

### PR DESCRIPTION
**Pull request by ichbtm**
hi here is a pull request to add dustLog (/sapi/v1/asset/dribblet) to the api

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ Following examples will use the `await` form, which requires some configuration
   - [assetDetail](#assetDetail)
   - [getBnbBurn](#getBnbBurn)
   - [setBnbBurn](#setBnbBurn)
+	- [dustLog](#dustlog)
```
Comment (balthazar) [inline README.md:99]: can you fix the indentation on this?

**Review by balthazar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ichbtm) [reply README.md]: done

**Comment by balthazar** [comment]
Thank you!

**Comment by ichbtm** [comment]
it was a pleasure ;)

**Pull request merged**